docs/guide/installation.rst

```diff
@@ -52,9 +47,44 @@ You can update to the latest version by running::
 
     conda update sunpy
 
+Testing sunpy
+=============
+
+sunpy provides a method to run the basic test suite that will check that the install has worked correctly.
+
+To do so, use the :func:`sunpy.self_test`::
+
+    import sunpy
+    sunpy.self_test()
+
+You will see something like the following in your terminal::
+
+    Starting sunpy self test...
+    Checking for packages needed to run sunpy:
+    All required and optional sunpy dependencies are installed.
+    Starting the sunpy test suite:
+    ...
+
+The tests will run and will report any fails which you can report at the `sunpy issue tracker <https://github.com/sunpy/sunpy/issues>`__ and we will strive to help.
+
+It is possible to run this command and in a situation where not all packages are installed in order to run the test suite, you will see::
+
+    Starting sunpy self test...
+    Checking for packages needed to run sunpy:
+    The following packages are not installed for the sunpy[database] requirement:
+    * sqlalchemy
+    ...
+    You do not have all the required dependencies installed to run the sunpy test suite.
+    If you want to run the sunpy tests install the 'tests' extra with `pip install sunpy[all,tests]`
+
+This should not appear under a conda install of sunpy but can under a pip install of sunpy.
```

Comment:
This is a little misleading. This talking about tests, but at the same time showing optional deps. It's actually quite likely that you will run into not having the tests deps installed.